src/fragments/lib/project-setup/flutter/platform-setup/android.mdx

```diff
@@ -1,48 +1,68 @@
-Amplify requires a minimum of API level 24 (Android 7.0), Gradle 7 and Kotlin > 1.9 when targeting Android.
+Amplify Flutter supports API level 24+ (Android 7.0+), and requires Gradle 8+, Kotlin 1.9+, and Java 17+ when targeting Android. Follow the steps below to apply these changes in your app. 
 
-_If your Flutter app was generated with Flutter 3.16 or lower, please follow the migration guide [here](https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply)._
+<Callout warning>
+The steps below are intended for Flutter apps created with Flutter version 3.16+. If your app was created prior to version 3.16, please follow the guide [here](https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply) to migrate to Gradle's declarative plugins block before following the steps below.
+</Callout>
 
-From your project root, navigate to the `android/` directory and open `settings.gradle` in the text editor of your choice. Update the Gradle plugin version to 7.4.2 or higher:
+1. Open `android/settings.gradle` and update the Android Gradle plugin and kotlin versions:
 
-```diff
-    plugins {
-        id "dev.flutter.flutter-plugin-loader" version "1.0.0"
--       id "com.android.application" version "7.3.0" apply false
-+       id "com.android.application" version "7.4.2" apply false
```

Comment:
Q: Should this be 8.1.0?

Reply:
This is the removed code. Diff might look confusing since this is a diff of a diff